win64.mak

```diff
@@ -17,8 +17,8 @@ IMPDIR=import
 
 MAKE=make
 
-DFLAGS=-m$(MODEL) -conf= -O -release -dip1000 -inline -w -Isrc -Iimport
-UDFLAGS=-m$(MODEL) -conf= -O -release -dip1000 -w -Isrc -Iimport
+DFLAGS=-m$(MODEL) -conf= -O -release -dip1000 -dip1008 -inline -w -Isrc -Iimport
+UDFLAGS=-m$(MODEL) -conf= -O -release -dip1000 -dip1008 -w -Isrc -Iimport
```

Comment:
Adding -dip1008 seems unrelated. There are also a couple more unrelated changes to win64.mak that cause build failures.